package.json

```diff
@@ -19,20 +19,18 @@
     "mobify"
   ],
   "authors": [
-    "Mobify",
-    "Kyle Peatt <kpeatt@gmail.com>"
+    "Mobify"
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/mobify/spline/issues"
   },
-  "dependencies": {
-    "grunt": "~0.4.4",
-    "grunt-contrib-sass": "~0.8.1"
-  },
   "devDependencies": {
-    "grunt": "~0.4.4",
+    "grunt": "^1.0.1",
```

Comment:
We should be pinning all dependencies here to specific versions to ensure maximum compatibility.